### ISSUE TITLE
MB Settings Page - wysiwyg issues

### ISSUE THREAD

**Issue by wgstjf** [rilwis/meta-box#971]
Hi guys,

I have a wysiwyg field in my MB Settings Page config that is within a cloneable group.  The editor shows as it should but the html is being converted on save so that I end up with visible html tags...any thoughts?

![image](https://cloud.githubusercontent.com/assets/3740694/18716725/b2a88f1a-8015-11e6-870a-6eb922d7dbad.png)

The code used to create the field is:

```php
// WYSIWYG - Author Bio
array(
	'name'    => __( 'Author Bio', 'indigo-options' ),
	'id'      => 'bio',
	'desc'  => __( 'Enter the text for this panel', 'indigo-metaboxes' ),
	'type'    => 'wysiwyg',
	'raw'     => false,
	'options' => array(
		'textarea_rows' => 3,
		'teeny'         => true,
		'media_buttons' => false,
	),
	'columns'	=>	'6',
),
```

Cheers,

Will

**Comment by wgstjf** [comment]
Hi guys,

Any thoughts on this??

Cheers,

Will

**Comment by rilwis** [comment]
Hi Will, just fixed this in Meta Box Group 1.1.6. Please update.
